docs/modules/misc/pages/integrations/spring-boot.adoc

```diff
@@ -8,14 +8,114 @@ It is available within this artifact:
 [source, xml, title="pom.xml", subs=attributes+]
 ----
 <dependencies>
-	<dependency>
-		<groupId>one.microstream</groupId>
-		<artifactId>microstream-integrations-spring-boot</artifactId>
-		<version>{maven-version}</version>
-	</dependency>
+   <dependency>
+      <groupId>one.microstream</groupId>
+      <artifactId>microstream-integrations-spring-boot</artifactId>
+      <version>{maven-version}</version>
+   </dependency>
 </dependencies>
 ----
 
-For more information consult the https://github.com/microstream-one/microstream/tree/master/integrations/spring-boot[readme] file.
-
 Official Spring Boot site: https://spring.io/projects/spring-boot[]
+
+The integration requires Spring Boot 2.x (with version 2.1.0.RELEASE being the minimal supported one).
+
+== Breaking changes
+
+Since version 8.0.0, the _StorageManager_ that is instantiated by the integration is also started where the previous version of the integration did not start the manager.  You can have the old behaviour of a _StorageManager_ that is not started by specifying the config value `one.microstream.auto-start=false`.
+
+== Configuration
+
+The configuration of the _StorageManager_ can be done using key/value pairs that are provided by Spring Boot external Configuration.  The configuration keys must be prefixed by `one.microstream`
+
+[source]
+----
+one.microstream.storage-directory=/opt/data/microstream
+one.microstream.channel-count=2
+----
+
+The list of all MicroStream configuration properties and their meaning are listed on our https://docs.microstream.one/manual/storage/configuration/properties.html[documentation page].
```

Comment:
No need to use the external URL for internal doc links. You can use [xrefs](https://docs.antora.org/antora/2.2/asciidoc/page-to-page-xref/) for that:
`xref:storage:configuration/properties.adoc[documentation page]`